src/Blockcore/Base/InitialBlockDownloadState.cs

```diff
@@ -53,13 +53,13 @@ public InitialBlockDownloadState(IChainState chainState, Network network, Consen
         /// <inheritdoc />
         public bool IsInitialBlockDownload()
         {
-            if (this.lastCheckpointHeight > this.chainState.ConsensusTip.Height)
+            if (this.lastCheckpointHeight > this.chainState.ConsensusTip?.Height)
```

Comment:
This method is called very heavily during IBD
I wonder if this is the best solution, the problem is its not set on startup right?
Maybe we should not let the UI call in to DI if node is not initialized

Reply:
The problem is just the UI uses this and it throws exception during debug running. It doesn't really show in the UI, but ends up stopping when debugging.

Reply:
Changes reverted back to original.